templates/terraform/examples/billing_budget_basic.tf.erb

```diff
@@ -0,0 +1,18 @@
+data "google_billing_account" "account" {
+  billing_account = "<%= ctx[:test_env_vars]['billing_acct'] -%>"
```

Comment:
This data source also wants `provider = google-beta`(and in the filter example)

Reply:
Done.